demos/kubernetes-helm/README.md

```diff
@@ -0,0 +1,106 @@
+# Install the Plugin using the Official Jenkins helm chart
+
+## Preparation
+
+Jenkins can be installed in Kubernetes using [`helm`](https://github.com/helm/helm).
+The latest stable helm chart can be found [`here`](https://github.com/helm/charts/tree/master/stable/jenkins).
+
+To have the latest Version of JCASC installed we need to enable Jenkins to make use of the [`Experimental Update Center`](https://jenkins.io/doc/developer/publishing/releasing-experimental-updates/).
+
+The [`official docker image`](https://github.com/jenkinsci/docker/blob/master/Dockerfile#L60) allows you to specify the experimental update center configuration using ENV variables:  
+
+```
+ENV JENKINS_UC https://updates.jenkins.io
+ENV JENKINS_UC_EXPERIMENTAL=https://updates.jenkins.io/experimental
+ENV JENKINS_INCREMENTALS_REPO_MIRROR=https://repo.jenkins-ci.org/incrementals
+```
+
+Now grab a copy of the helm chart [`values file`](https://github.com/helm/charts/blob/master/stable/jenkins/values.yaml) and adjust the Master part a little bit:
+
+```yaml
+Master:
+  Name: jenkins-master
+  Image: "jenkins/jenkins"
+  ImageTag: "2.134-alpine"
+  ImagePullPolicy: "IfNotPresent"
+  Component: "jenkins-master"
+  UseSecurity: true
+  AdminUser: admin
+  AdminPassword: "formetoknowforyoutofindout"
+  Cpu: "200m"
+  Memory: "1024Mi"
+  ContainerEnv:
+    - name: JENKINS_UC
+      value: https://updates.jenkins.io
+    - name: JENKINS_UC_EXPERIMENTAL
+      value: https://updates.jenkins.io/experimental
+    - name: JENKINS_INCREMENTALS_REPO_MIRROR
+      value: https://repo.jenkins-ci.org/incrementals
+  InitContainerEnv:
+    - name: JENKINS_UC
+      value: https://updates.jenkins.io
+    - name: JENKINS_UC_EXPERIMENTAL
+      value: https://updates.jenkins.io/experimental
+    - name: JENKINS_INCREMENTALS_REPO_MIRROR
+      value: https://repo.jenkins-ci.org/incrementals
+
+  # List of plugins to be install during Jenkins master start
+  # mind the last plugin in the list now ;)
+  InstallPlugins:
+      - kubernetes:latest
+      - kubernetes-credentials:latest
+      - workflow-aggregator:latest
+      - workflow-job:latest
+      - credentials-binding:latest
+      - git:latest
+      - configuration-as-code:0.10-alpha            
+```
+
+## Installations
+
+Now, deploy the Helm chart with those customized values:
+
+```
+helm install --name jenkins stable/jenkins \
+-f values.yml
```

Comment:
Can you make the whole snippet as one-line? It makes the Markdown file look better when scrolling.

I see at least no added value on keeping it in a separate line.

Reply:
done